tests/Fixture/composer.json

```diff
@@ -11,7 +11,7 @@
         }
     ],
     "require": {
-        "php": ">=5.4.16",
+        "php": ">=5.6",
         "cakephp/cakephp": "~3.0",
```

Comment:
Cake version also needs to be updated. I don't remember off hand in which minor we bumped up to PHP 5.6.